Description: Set SONAME using -release as it is not stable from upstream.
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ndpi/src/lib/Makefile.am
===================================================================
--- ndpi.orig/src/lib/Makefile.am
+++ ndpi/src/lib/Makefile.am
@@ -8,16 +8,11 @@ noinst_HEADERS = ../include/ndpi_main.h
 		../include/linux_compat.h
 
 
-# this will create a library 1.4.0
-LIB_AC=1
-LIB_REV=4
-LIB_ANC=0
-
 # ntop
 AM_CFLAGS=-fPIC @PLATFORM_DEPENDENT@
 
 libndpi_la_CPPFLAGS = -I$(top_srcdir)/src/include/  -I$(top_srcdir)/src/lib/third_party/include/
-libndpi_la_LDFLAGS=-version-info ${LIB_AC}:${LIB_REV}:${LIB_ANC}
+libndpi_la_LDFLAGS=-release @VERSION@
 
 libndpi_la_SOURCES = \
 			third_party/src/ahocorasick.c \
